src/fr/auth.php

```diff
@@ -1,17 +1,6 @@
 <?php
 
 return [
-    /*
```

Comment:
You delete comments ?

Reply:
I [said in the discussion](https://github.com/Laravel-Lang/lang/discussions/1555) that in this case all comments will be automatically deleted. The code works with an array, and does not parse strings.

Reply:
For a long time, the Russian-speaking community has not paid attention to the comments inside the translation files, since the name of the file already makes it clear what messages are inside.